start/src/main/java/org/apache/accumulo/start/classloader/AccumuloClassLoader.java

```diff
@@ -44,21 +44,21 @@
 
 public class AccumuloClassLoader {
 
-  public static final String CLASSPATH_PROPERTY_NAME = "general.classpaths";
+  public static final String GENERAL_CLASSPATHS = "general.classpaths";
   public static final String MAVEN_PROJECT_BASEDIR_PROPERTY_NAME = "general.maven.project.basedir";
   public static final String DEFAULT_MAVEN_PROJECT_BASEDIR_VALUE = "";
 
-  private static String SITE_CONF;
+  private static URL accumuloConfigUrl;
   private static URLClassLoader classloader;
   private static final Logger log = LoggerFactory.getLogger(AccumuloClassLoader.class);
 
   static {
-    String configFile = System.getProperty("org.apache.accumulo.config.file", "accumulo-site.xml");
-    if (System.getenv("ACCUMULO_CONF_DIR") != null) {
-      // accumulo conf dir should be set
-      SITE_CONF = System.getenv("ACCUMULO_CONF_DIR") + "/" + configFile;
+    String configFilename = System.getProperty("org.apache.accumulo.config.file", "accumulo-site.xml");
+    accumuloConfigUrl = AccumuloClassLoader.class.getClassLoader().getResource(configFilename);
+    if (accumuloConfigUrl == null) {
+      log.warn("Failed to load Accumulo configuration '{}' from classpath", configFilename);
```

Comment:
> it would be nice if somehow users could set an absolute path to a configuration file on the file system.

Any reason to not make it look like a URI and only accept "file://"?